raspbian/applications/README.md

```diff
@@ -4,3 +4,7 @@
     - Information on the demonstration camera software
 - [OMXPlayer](omxplayer.md)
     - Information on the OMXPlayer command line media player
+- [Minecraft for pi]
+    - 3D block based building platform with its own simple 
+      line and command coding and also teaches electronic
+      circuitry.
```

Comment:
@cntrlANC Just out of curiosity, how does Minecraft teach electronic circuitry?

Reply:
Probably referring to redstone, but I don't think that works in the pi edition.

Reply:
yes.

Reply:
I can confirm that there is support for redstone as it is a more polished version. Minecraft pi edition also has support for extended programming languages.